subgraph/subgraph.yaml

```diff
@@ -134,3 +134,49 @@ dataSources:
         - event: UpdatePlatformFeeRecipient(address)
           handler: handleUpdatePlatformFeeRecipient
       file: ./src/DigitalaxAuction.mapping.ts
+  - kind: ethereum/contract
+    name: DigitalaxGarmentCollection
+    network: 'rinkeby'
+    source:
+      address: '0x9f3077F6062Ab74c2d7D79885Be4744F8a839E93'
+      abi: DigitalaxGarmentCollection
+      startBlock: 
+    mapping:
+      kind: ethereum/events
+      apiVersion: 0.0.4
+      language: wasm/assemblyscript
+      entities:
+        - MintGarmentCollection
+      abis:
+        - name: DigitalaxGarmentCollection
+          file: ./abis/DigitalaxGarmentCollection.json
+      eventHandlers:
+        - event: MintGarmentCollection(uint256)
+          handler: handleGarmentCollectionMinted
+      file: ./src/DigitalaxGarmentCollection.mapping.ts
+  - kind: ethereum/contract
+    name: DigitalaxMarketplace
+    network: 'rinkeby'
+    source:
+      address: '0x837CA13b6E6Ea4cb3ccbDBBE4EF7e49E40305F3d'
+      abi: DigitalaxMarketplace
+      startBlock: 7827826
+    mapping:
+      kind: ethereum/events
+      apiVersion: 0.0.4
+      language: wasm/assemblyscript
+      entities:
+        - OfferCreated
+        - UpdateOfferPrimarySalePrice
+        - OfferPurchased
+      abis:
+        - name: DigitalaxMarketplace
+          file: ./abis/DigitalaxMarketplace.json
+      eventHandlers:
+        - event: OfferCreated(indexed uint256)
```

Comment:
**We might want to handle some other events like**

  ``` 
  event OfferCancelled(
        uint256 indexed garmentTokenId
    );
 ```
    
The auction handles some events like this, so you should also include:

 
    ```
        event UpdateMarketplacePlatformFee(
        uint256 platformFee
    );
    event UpdateMarketplaceDiscountToPayInErc20(
        uint256 discount
    );
     event UpdatePlatformFeeRecipient(
        address payable platformFeeRecipient
    );
    
    ```
    
  But I am not sure if it is worth keeping track of the platform fee recipient and access controls, not sure what the industry standard would be in terms of whether every event in the contract is necessary.
    ```
event UpdateAccessControls(
        address indexed accessControls
    );
    ```
    
Just want to make sure we dont have problems later on by including as much as possible